### PR TITLE
Feat/clean pipelines

### PR DESCRIPTION
**Pull request by guillermogutierrez**
Remove unnecessary files

### CODE REVIEW

**Review by radoslaw-orlowski — APPROVED** [review]

File: build/azDevOps/azure/azuredevops-vars.yml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  - name: role
+    value: backend
+  #
+  # SelfConfig
+  # If you haven't specified source_repo at cli runtime please ensure you replace it here
+  # It is case sensitive for TFS based repos
+
+  - name: self_remote_repo
+    value: "amido/$(self_repo)"
+  # - name: self_repo_src
```
Comment (johnsall) [inline build/azDevOps/azure/azuredevops-vars.yml:27]: clean this

File: build/azDevOps/azure/azuredevops-vars.yml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  - name: yamllint_scan_directory
+    value: "."
+
+  # Build Task Naming
+  - name: java_project_type
+    value: "Java App"
+  - name: functional_test_project_type
+    value: "Functional API Tests"
+
+  # Terraform Configuration
```
Comment (johnsall) [inline build/azDevOps/azure/azuredevops-vars.yml]: and this

File: pom.xml [hunk 3]
```diff
@@ -62,13 +63,13 @@
...
         </dependency>
 
-<!--        &lt;!&ndash; this should be shared only as long as stacks supports only azure &ndash;&gt;-->
-<!--        <dependency>-->
-<!--            <groupId>com.microsoft.azure</groupId>-->
-<!--            <artifactId>applicationinsights-logging-logback</artifactId>-->
-<!--            <version>${applicationinsights.version}</version>-->
-<!--            <scope>runtime</scope>-->
-<!--        </dependency>-->
+        <!--        &lt;!&ndash; this should be shared only as long as stacks supports only azure &ndash;&gt;-->
```
Comment (johnsall) [inline pom.xml]: if we don't need this let's remove it

File: build/azDevOps/azure/azuredevops-vars.yml (hunk 1 above)
Comment (johnsall) [reply build/azDevOps/azure/azuredevops-vars.yml]: just this one left :P

**Review by johnsall — APPROVED** [review]
lgtm

**Pull request merged**